invenio_rdm_records/services/communities/service.py

```diff
@@ -205,10 +203,22 @@ def _remove(self, identity, community_id, record):
         if community_id not in record.parent.communities.ids:
             raise RecordCommunityMissing(record.id, community_id)
 
-        # check permission here, per community: curator cannot remove another community
-        self.require_permission(
+        # If config is true and there is only 1 communities left to remove
+        is_community_required = current_app.config["RDM_RECORD_ALWAYS_IN_COMMUNITY"]
+        is_last_community = len(record.parent.communities.ids) == 1
+        # Then, check for permissions to remove last community
+        can_remove_last_community = self.check_permission(
```

Comment:
the name of this variable is misleading, because you don't use is really only for checking last community permission, but in general, if the user can remove community
```suggestion
        can_remove_community = self.check_permission(
```

Reply:
The check for removing a community is performed by permissions, here we are changing the permission check based on the config. So if the uploader has permission to remove the community, he won't be able to remove just the last community since a record should always have a community if the config is set to true.